src/block/block_open.c

```diff
@@ -194,41 +196,23 @@ __wt_block_open(WT_SESSION_IMPL *session,
 	/* Configuration: optional OS buffer cache maximum size. */
 	WT_ERR(__wt_config_gets(session, cfg, "os_cache_max", &cval));
 	block->os_cache_max = (size_t)cval.val;
-#ifdef HAVE_POSIX_FADVISE
```

Comment:
@agorrod

> Why did you decide to drop this error checking? If we are going to start silently ignoring this, we should probably update our documentation to reflect that this is an advisory setting.
 
The change was in 9240bb3. At the time, I wrote:

> Remove block manager handle open tests of cache-max and dirty-cache-max
configuration conflicts with specifying direct I/O, lacking posix_fadvise
and lacking sync_file_range. We can't know in the block manager what the
underlying layer can do, and also, the old code was wrong, direct I/O is
set on a file type basis, so testing against a specific handle open isn't
correct.

In short, with pluggable file systems, we don't know until we see the ENOTSUP whether or not any particular call is going to work.

We could pretty easily output an message the first time one of these calls fails with ENOTSUP, if that's better.